Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaILogger.cs

```diff
@@ -9,29 +10,26 @@ internal class LambdaILogger : ILogger
         private readonly string _categoryName;
         private readonly LambdaLoggerOptions _options;
 
-        // Constants
-        private const string DEFAULT_CATEGORY_NAME = "Default";
+
+        internal IExternalScopeProvider ScopeProvider { get; set; }
 
         // Constructor
         public LambdaILogger(string categoryName, LambdaLoggerOptions options)
         {
-            _categoryName = string.IsNullOrEmpty(categoryName) ? DEFAULT_CATEGORY_NAME : categoryName;
+            _categoryName = categoryName;
             _options = options;
         }
 
         // ILogger methods
-        public IDisposable BeginScope<TState>(TState state)
-        {
-            // No support for scopes at this point
-            // https://docs.asp.net/en/latest/fundamentals/logging.html#scopes
-            return new NoOpDisposable();
-        }
+        public IDisposable BeginScope<TState>(TState state) => ScopeProvider?.Push(state) ?? NullScope.Instance;

```

Comment:
Instead of using NullScope from ASP.NET Core internal namespace we should continue to use the existing NoOpDisposable object

Reply:
ok. I'll bring back NoOpDisposable